src/api/OrdersApi.js

```diff
@@ -104,19 +103,14 @@ export default class OrdersApi {
   placeOrderWithHttpInfo(id, opts) {
     opts = opts || {};
 
-    const _placeOrderRequest = PlaceOrderRequest.constructFromObject(
-      opts,
-      new CreateOrderRequest()
-    );
-    let postBody = _placeOrderRequest;
-
     // verify the required parameter 'id' is set
     if (id === undefined || id === null) {
       throw new Error(
         "Missing the required parameter 'id' when calling placeOrder"
       );
     }
 
+    let postBody = opts;
```

Comment:
For optional parameters, this generation process is simpler. It doesn't validate the parameter as well, but the tradeoff on the template side are worth it.